File: scripts/bench_fib.lua
local function fib(n)
    if n <= 1 then return n end
    return fib(n - 1) + fib(n - 2)
end

local t0 = os.clock()
local result = fib(35)
local t1 = os.clock()
print(string.format("lua    fib(35) = %d  time: %.4fs", result, t1 - t0))

File: scripts/bench_fib.py
import time
def fib(n):
    if n <= 1: return n
    return fib(n-1) + fib(n-2)
t0 = time.perf_counter()
r = fib(35)
t1 = time.perf_counter()
print(f"python fib(35) = {r}  time: {t1-t0:.4f}s")
